makefiles/master/rules.mk

```diff
@@ -46,6 +46,10 @@ ifneq ($(SYSROOT),)
 ifneq ($(call __exists,$(SYSROOT)),$(_THEOS_TRUE))
 	$(ERROR_BEGIN) "Your current SYSROOT, “$(SYSROOT)”, appears to be missing." $(ERROR_END)
 endif
+# SYSROOT existence check failed so
+# it's now blank and we're bailing out
+else
+	$(ERROR_BEGIN) "Your current SYSROOT does not exist." $(ERROR_END)
```

Comment:
Since the error case is that we didn't find any sysroot at all (user didn't set SYSROOT directly, and darwin makefiles didn't find one in Xcode or $THEOS/sdks/), I think this should be reworded to be more clear about the problem and provide a link to the docs for installing an SDK.